Libs/Reflex/src/main/java/reflex/node/SubNode.java

```diff
@@ -66,6 +67,9 @@ public ReflexValue evaluate(IReflexDebugger debugger, Scope scope) {
             ret = new ReflexValue(a.asDate().sub(b.asInt()));
         } else if (a.isList()) {
             List<ReflexValue> list = a.asList();
+            if (!list.contains(b)) {
```

Comment:
Is this the behavior we want? That if you try to remove something from a list and the list does not contain the item we throw an exception? Not all languages have *that* behavior. Some of them do "remove if exist". Otherwise all of our code will probably now say

if x in y do
   remove(y,x);
end